gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -1020,8 +1020,14 @@ public class ConfigurationKeys {
    * Configuration properties related to flowGraphs
    */
 
-  public static final String JAVA_PROPS_EXTENSIONS = "javaPropsExtensions";
-  public static final String HOCON_FILE_EXTENSIONS = "hoconFileExtensions";
+  public static final String JAVA_PROPS_EXTENSIONS = "flowGraph.javaPropsExtensions";
+  public static final String HOCON_FILE_EXTENSIONS = "flowGraph.hoconFileExtensions";
```

Comment:
Yes it does seem like they need to be, they use different pullers under the hood so this is needed to overwrite their differentiations. I'll add comments though. I think in the future we'd wan to move most of the flowgraph if not all of it to hocon format just because it's richer.